relay-metrics/src/aggregation.rs

```diff
@@ -874,19 +874,19 @@ impl From<AggregateMetricsErrorKind> for AggregateMetricsError {
 enum AggregateMetricsErrorKind {
     /// A metric bucket had invalid characters in the metric name.
     #[error("found invalid characters")]
-    InvalidCharacters,
+    InvalidCharacters(String),
```

Comment:
in each of this cases you add, you might want to update also the macro like `#[error("found invalid characters: ({0})")]` to actually print the error with the wrong value. 

What do you think?

Reply:
I was wondering how the new error message would be printed.

Reply:
Good one, that's what I actually wanted.